test/.env.test.example

```diff
@@ -65,3 +65,8 @@ DB_SEEDS_DIR="./src/database/seeds"
 # Auth0
 #
 AUTH0_HOST="http://localhost:3333"
+
+#
+# PriceTicker
+#
+DATA_CHECK_DELAY=0.05
```

Comment:
change this to seconds

Reply:
private async checkDiffBtwDate(timestamp: Date): Promise<number> {
        const current: any = new Date();
        const ticker: any = new Date(timestamp);
        return (current - ticker) / 1000;
    }

now this functions return in seconds